dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -178,8 +172,8 @@ public void testQuery() throws Exception {
         List<SubSnpCoreFields> readSnps = readAll(reader);
 
         assertEquals(21, readSnps.size());
-        for(SubSnpCoreFields expectedSnp : expectedSubsnps) {
-            assertContains(readSnps, expectedSnp);
+        for (SubSnpCoreFields expectedSnp : expectedSubsnps) {
+            assertTrue(readSnps.contains(expectedSnp));
```

Comment:
@jmmut can you comment on this? You created `assertContains` for convenience, but it only adds a debug message so I wonder if we could be overdoing it. @pabarcgar clearly thought so 😛

Reply:
I made it because with `assertTrue(readSnps.contains(expectedSnp))`, you can not see anything in case the test fails, it could be that the element is not there, or that it is there but some attribute changed. As the test is checking that all the transformations are correct, in the case that the test fails, you have to debug to see which attribute transformation failed. With the asserContains you can see the collection, the element, and understand why the element is not in the collection without running the test again.